Makefile

```diff
@@ -31,7 +31,7 @@ ICONNAME   = nanos_app_hathor.gif
 APPVERSION = 0.0.1
 
 # The --path argument here restricts which BIP32 paths the app is allowed to derive.
-APP_LOAD_PARAMS = --appFlags 0x40 --path "44'/93'" --curve secp256k1 --curve ed25519 $(COMMON_LOAD_PARAMS)
+APP_LOAD_PARAMS = --appFlags 0x40 --path "44'/280'" --curve secp256k1 $(COMMON_LOAD_PARAMS)
```

Comment:
Where is it used? I think I've seen the path in a different file. I'm just concerned about duplicate values (so, multiple places to update if needed).

Reply:
It's used when we derive the keys, but in those cases we have the array, not the string.

Reply:
I also changed this on the other PR, where the bip44 path is on a single file. `280` is now defined in the Makefile only.